jwst/pipeline/calwebb_tso3.cfg

```diff
@@ -8,6 +8,4 @@ scale_detection = False
         config_file = outlier_detection_tso.cfg
```

Comment:
`scale_detection = False` above can probably be removed, as it is set in the pipeline `spec`

Reply:
Whoops, missed that one. Good catch!  Will remove.

Reply:
Done.